### DISCUSSION TITLE
Farm Bundle Analyzer

### DISCUSSION THREAD

**Discussion by remloyal** [farm-fe/farm#1794] [Ideas]
Does it support package analysis after farm build?

The command line is similar to `farm build --analyze`
refer to [Bundle analysis](https://rspack.dev/guide/optimization/analysis)

Or in a form similar to a plugin：
refer to [webpack-bundle-analyzer](https://www.npmjs.com/package/webpack-bundle-analyzer)

Or can it be achieved through other means?

**Comment by wre232114** [comment]
try https://www.npmjs.com/package/@farmfe/js-plugin-visualizer

**Reply by remloyal** [reply]
> 试用 https://www.npmjs.com/package/@farmfe/js-plugin-visualizer

Thank you very much, but there seems to be an issue with the document you wrote. It's not `pnpm add - D @ farmfe/visualizer`, but `@farmfe/js-plugin-visualizer`

**Reply by wre232114** [reply]
I'll fix it
